wgpu/tests/buffer.rs

```diff
@@ -0,0 +1,19 @@
+use crate::common::{initialize_test, TestParameters};
```

Comment:
Great use of a test, all I would say is that this should probably be a file for all pieces of attached resource data.

Reply:
I'm sorry, I don't entirely understand what change you're proposing. Do you mean renaming this module to something more general, like `resource.rs`?

Reply:
Yeah or `resource_descriptor_accessor` or something that captures that vibe.

Reply:
I'm skeptical that this is a good way to organize tests, but okay.